### ISSUE TITLE
pos_category_multi incompatible with pos reporting

### ISSUE THREAD

**Issue by bealdav** [itpp-labs/pos-addons#1176]
## What's wrong

Here is the report with an runbot instance

![Screenshot from 2019-11-25 10-02-27](https://user-images.githubusercontent.com/1853434/69527001-ac02bf80-0f6b-11ea-8375-27aad87b6a31.png)

When you populate pos_categ_ids, the report goes wrong

## How it could be better

An onchange could populate pos_categ_id from pos_categ_ids with the first value.

What do you think @yelizariev  ?

side note: my customer use pos_category_multi as paid module.

**Comment by yelizariev** [comment]
The description is missing :smiley_cat:

**Comment by Paul-LFG** [comment]
I canot use the report as i'm using pos_category_multi, the report mixup all the articles in the categories and its not trustfull. 
Have a good reporting is mandatory, so the usage of pos_category_multi must take care of this Reporting issue.    Hope that can be solved asap.

**Comment by Paul-LFG** [comment]
Of course, when my company LFG buy this module 6 months ago, it was expected to have solid reporting.  Unfortunately it didn't work as it should for this purpose.

**Comment by bealdav** [comment]
Hi @yelizariev, please give us you opinion on these questions:

1/ do you consider the described problem as real ? 
2/ do you consider than the proposed solution is generic for your module and is the best answer or there are any other proposals to consider ?

**Comment by bealdav** [comment]
Hi @yelizariev.
Have you any news ? Thanks
